web-search-engines/langchain4j-web-search-engine-searchapi/src/main/java/dev/langchain4j/web/search/searchapi/SearchApiWebSearchApi.java

```diff
@@ -0,0 +1,15 @@
+package dev.langchain4j.web.search.searchapi;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Header;
+import retrofit2.http.QueryMap;
+
+import java.util.Map;
+
+interface SearchApiWebSearchApi {
+
+    @GET("/api/v1/search")
```

Comment:
```suggestion
    @GET("api/v1/search")
```
I have learned recently that when path starts with `/` in retrofit, it means absolute path and any sub-path in `baseUrl` will be ignored:

`baseUrl("http://example.com/sub/")` + `@GET("/api/v1/search")` -> `http://example.com/api/v1/search` (`sub` is ignored)
`baseUrl("http://example.com/sub/")` + `@GET("api/v1/search")` -> `http://example.com/sub/api/v1/search`